averages.js

```diff
@@ -0,0 +1,45 @@
+function total(nums) {
+  return nums.reduce(function(sum, value) {
```

Comment:
Yay reduce! Do you know how you could make the initial value start at, for example, 5 or -1?